[2024-01-28T16:28:39.357+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-22T00:00:00+00:00 [queued]>
[2024-01-28T16:28:39.386+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-22T00:00:00+00:00 [queued]>
[2024-01-28T16:28:39.390+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-28T16:28:39.621+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2023-03-22 00:00:00+00:00
[2024-01-28T16:28:39.632+0000] {standard_task_runner.py:60} INFO - Started process 4510 to run task
[2024-01-28T16:28:39.660+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2023-03-22T00:00:00+00:00', '--job-id', '1515', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmpgrjypvkm']
[2024-01-28T16:28:39.690+0000] {standard_task_runner.py:88} INFO - Job 1515: Subtask get_stackoverflow_data
[2024-01-28T16:28:39.860+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-22T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-28T16:28:40.228+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2023-03-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-22T00:00:00+00:00'
[2024-01-28T16:28:43.007+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-28T16:28:43.039+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20230322T000000, start_date=20240128T162839, end_date=20240128T162843
[2024-01-28T16:28:43.164+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-28T16:28:43.282+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
